activemodel/lib/active_model/validations/exclusion.rb

```diff
@@ -29,7 +29,9 @@ module HelperMethods
       # Configuration options:
       # * <tt>:in</tt> - An enumerable object of items that the value shouldn't
       #   be part of. This can be supplied as a proc, lambda or symbol which returns an
-      #   enumerable. If the enumerable is a range the test is performed with
+      #   enumerable. If the enumerable is a numerical, time or datetime range the test
+      #   is performed with <tt>Range#cover?</tt>, otherwise with <tt>include?</tt>. When
```

Comment:
There is a warning box with:
> Using `+...+` for fixed-width font only works with simple content like ordinary method names, symbols, paths (with forward slashes), etc. Please use `<tt>...</tt>` for everything else, notably class or module names with a namespace as in `<tt>ActiveRecord::Base</tt>`.

Since you added the text, I would go with `+:include?+`